docs/api/1.x/timestamps.rst

```diff
@@ -81,21 +81,28 @@ the ``If-None-Match: *`` request header.
-| Not changed                 | Create      | Overwrite    | Modify        | Delete        |
-+-----------------------------+-------------+--------------+---------------+---------------+
-|| **If-None-Match: ***                                                                    |
-+-----------------------------+-------------+--------------+---------------+---------------+
-| Id exists                   | ``HTTP 412``| ``HTTP 412`` | No effect     | No effect     |
-+-----------------------------+-------------+--------------+---------------+---------------+
-| Id unknown                  | Create      | Create       | No effect     | No effect     |
-+-----------------------------+-------------+--------------+---------------+---------------+
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+|                             | GET         | POST        | PUT         | PATCH       | DELETE      |
++=============================+=============+=============+=============+=============+=============+
+|| **If-Match: "timestamp"**                                                                        |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Not changed                 | Fetch       | Fetch       | Overwrite   | Modify      | Delete      |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Changed meanwhile           | ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``|
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+|| **If-Match: ***                                                                                  |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id exists                   | Fetch       | Fetch       | Overwrite   | Modify      | Delete      |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id unknown                  | ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``|
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+|| **If-None-Match: ***                                                                             |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id exists                   | ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``|
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id unknown                  | No effect   | Create      | Create      | No effect   | No effect   |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+
```

Comment:
We could mention those changes in the API changelog (and increment the HTTP API number maybe)